spec/controllers/account/admin/article_search_spec.rb

```diff
@@ -1,20 +1,26 @@
 require 'rails_helper'
 
 RSpec.describe Account::Admin::ArticleController, type: :controller do
+  after do
```

Comment:
https://makandracards.com/makandra/46867-rspec-around-all-and-around-each-hook-execution-order